specification/monitor/resource-manager/Microsoft.Monitor/preview/2021-06-03-preview/monitoringAccounts_API.json

```diff
@@ -0,0 +1,556 @@
+        }
+      }
+    },
+    "IngestionSettings": {
+      "description": "Settings for data ingestion",
+      "type": "object",
+      "properties": {
+        "dataCollectionRule": {
+          "description": "The Azure resource Id of the default data collection rule for this monitoring account.",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        },
+        "dataCollectionEndpoint": {
+          "description": "The Azure resource Id of the default data collection endpoint for this monitoring account.",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        }
+      }
+    },
+    "MonitoringAccount": {
+      "description": "Properties of a monitoring account",
+      "type": "object",
+      "properties": {
+        "accountId": {
+          "description": "The immutable ID of the monitoring account. This property is read-only.",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        },
+        "metrics": {
+          "description": "Information about metrics for the monitoring account",
+          "allOf": [
+            {
+              "$ref": "#/definitions/Metrics"
+            }
+          ],
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        },
+        "provisioningState": {
+          "description": "The provisioning state of the monitoring account. Set to Succeeded if everything is healthy.",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        },
+        "defaultIngestionSettings": {
+          "description": "The Data Collection Rule and Endpoint used for ingestion by default.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/IngestionSettings"
+            }
+          ],
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        }
+      }
+    },
+    "MonitoringAccountResource": {
+      "description": "A Monitoring Account definition",
+      "required": [
+        "location"
+      ],
+      "type": "object",
+      "properties": {
+        "properties": {
+          "description": "Resource properties",
+          "allOf": [
+            {
+              "$ref": "#/definitions/MonitoringAccount"
+            }
+          ],
+          "x-ms-client-flatten": true
+        },
+        "location": {
+          "description": "The region where the resource lives",
+          "type": "string",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "tags": {
+          "description": "Resource tags",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "id": {
```

Comment:
cosnider using trackedresource from commontypes for this

Reply:
Thanks for pointing me at that, makes this look much simpler